.vscode/settings.json

```diff
@@ -9,5 +9,8 @@
   },
   "[json]": {
     "editor.defaultFormatter": "dprint.dprint"
+  },
+  "editor.codeActionsOnSave": {
```

Comment:
Are there any strong opinions on adding this? I can add this to my user-settings instead

Reply:
I generally lean against forced on save actions. @mfedderly thoughts?

Reply:
I'm generally pro having robots do stuff for me. We have to pass eslint to get code merged anyhow so we should at least do this as a precommit hook. We can also turn this setting off super easily later if we don't like it.